src/components/PageLayout.js

```diff
@@ -127,46 +126,44 @@ const ResetStyles = (
 );
 
 const PageLayout = (props) => {
-  const mainRef = React.useRef();
+  // const mainRef = React.useRef();
 
-  const [isCookieSettingsOpen, setIsCookieSettingsOpen] = React.useState(
-    !config.getSetting(TRACKING_STORAGE_KEY, 'trackingStateChosen')
-  );
+  // const [isCookieSettingsOpen, setIsCookieSettingsOpen] = React.useState(
+  //   !config.getSetting(TRACKING_STORAGE_KEY, 'trackingStateChosen')
+  // );
 
-  React.useEffect(() => {
-    if (mainRef.current && isCookieSettingsOpen) {
-      mainRef.current.scrollTop = 0;
-    }
-  }, [mainRef, isCookieSettingsOpen]);
+  // React.useEffect(() => {
+  //   if (mainRef.current && isCookieSettingsOpen) {
+  //     mainRef.current.scrollTop = 0;
+  //   }
+  // }, [mainRef, isCookieSettingsOpen]);
```

Comment:
Typically wouldn't leave comments in a PR, but felt useful to leave this stuff?

Reply:
probably not useful anymore but we can remove later